geometry/pair.hpp

```diff
@@ -108,6 +108,9 @@ using enable_if_vector_t = typename enable_if_vector<T, U>::type;
 template<typename T1, typename T2>
 class Pair final {
  public:
+  Pair()
+    requires std::default_initializable<T1> && std::default_initializable<T2> =
+      default;
```

Comment:
Would this fit at the end of the previous line?

Reply:
No.